<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiGuider</class>
 <widget class="QMainWindow" name="PiGuider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>20</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>53</red>
        <green>10</green>
        <blue>10</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>20</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>53</red>
        <green>10</green>
        <blue>10</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>20</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>53</red>
        <green>10</green>
        <blue>10</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>16</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>PiGuider</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget
{
	background-color: rgb(30,0,0);
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exitButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#exitButton
{
	background-color: transparent;
	background: none;
	background-repeat: none;
	border-image: url(:/media/icons/tools-512x512/shutdown.png);
}
#exitButton:pressed
{
	border-image: url(:/media/icons/tools-512x512/shutdown-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mainImageLabel">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>478</width>
      <height>478</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainImageLabel
{
    border: 2px solid rgb(100, 0, 0);
    border-radius: 4px;
    padding: 2px;
/*	background-image:  url(:/media/NightSky.png);*/
	background-color: rgb(0,0,10);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="guideButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#guideButton
{
	background-color: transparent;
	background: none;
	background-repeat: none;
	border-image: url(:/media/icons/sections-512x512/Guide.png);
}
#guideButton:pressed
{
	border-image: url(:/media/icons/sections-512x512/Guide-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="focusButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#focusButton
{
	background-color: transparent;
	background: none;
	background-repeat: none;
	border-image: url(:/media/icons/sections-512x512/ManualFocus.png);
}
#focusButton:pressed
{
	border-image: url(:/media/icons/sections-512x512/ManualFocus-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
	background-color: rgb(60,0,0);
	color: rgb(0,180,0);
}</string>
    </property>
    <property name="text">
     <string>BLINK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intervalometerSoftButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#intervalometerSoftButton
{
	background-color: transparent;
	background: none;
	background-repeat: none;
	border-image: url(:/media/icons/sections-512x512/SoftIntervalometer.png);
}
#intervalometerSoftButton:pressed
{
	border-image: url(:/media/icons/sections-512x512/SoftIntervalometer-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="intervalometerHardButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#intervalometerHardButton
{
	background-color: transparent;
	background: none;
	background-repeat: none;
	border-image: url(:/media/icons/sections-512x512/HardIntervalometer.png);
}
#intervalometerHardButton:pressed
{
	border-image: url(:/media/icons/sections-512x512/HardIntervalometer-pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledLabelGuider">
    <property name="geometry">
     <rect>
      <x>573</x>
      <y>44</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledLabelDSLR">
    <property name="geometry">
     <rect>
      <x>704</x>
      <y>38</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledLabelSoftIntervalometer">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>115</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledLabelHardIntervalometer">
    <property name="geometry">
     <rect>
      <x>697</x>
      <y>117</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
